src/main/cpp/t06_quad.cpp

```diff
@@ -17,9 +17,27 @@
 #include "t06_quad.h"
 #include <iostream>
 #include <cmath>
+#include <math.h>
 
 using namespace std;
 
 int t06_quad() {
-
+    double a,b,c,D,x1,x2;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'x2' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4353647)